bikespace_dashboard/js/components/map.js

```diff
@@ -51,7 +63,23 @@ class Map extends Component {
     L.tileLayer(DEFAULT_TILES.url, {
       attribution: DEFAULT_TILES.attribution,
     }).addTo(this.lmap);
-    
+    L.control
+      .locate({
+        flyTo: true,
+        onLocationError: err => {
+          const message =
+            CUSTOM_GEO_ERROR_MESSAGES[err.code] ||
+            'Unknown error while trying to locate you';
+          this.analytics_event('locationerror', {code: err.code, message});
```

Comment:
@sonalranjit @tallcoleman thought it might be a good idea to add analytics for both `locationerror` and `locationfound` (see below) events. Helps us find out the number of people that request location and discover potential problems with the location UX